docs/source/deep-learning/tensorflow/guide/index.rst

```diff
@@ -0,0 +1,269 @@
+Tensorflow within MLflow
+=========================
+
+In this guide we will walk you through how to use Tensorflow within MLflow. We will demonstrate
+how to track your Tensorflow experiments and log your Tensorflow models to MLflow.
+
+Autologging Tensorflow Experiments
+-----------------------------------
+
+.. attention::
+    Autologging is only supported when you are using the ``model.fit()`` Keras API to train
+    the model. Additionally only Tensorflow >= 2.3.0 is supported. If you are using an older version
+    of Tensorflow or Tensorflow without Keras, please use manual logging.
+
+MLflow can automatically log metrics and parameters from your Tensorflow training. To enable
+autologging, simply run :py:func:`mlflow.tensorflow.autolog()`:
```

Comment:
(or just mlflow.autolog()). We might want to mention the two ways of enabling it